api_docs/configuring-python-bindings.md

```diff
@@ -8,12 +8,62 @@ using Zulip's [bot framework](/api/writing-bots).
 In order to use them, you need to configure them with your API key and other
 settings. There are two ways to achieve that:
 
- - With a file called `.zuliprc`, located in your home directory.
+ - With a file called `zuliprc` passed as an argument via the `--config-file`
```

Comment:
This might read better as "With a `zuliprc` file passed ..."

Reply:
Sounds good to me.